### ISSUE TITLE
Fix `invalid-attr` rule error when using `@` shorthand event handler with modifier on Vue

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#783]
- Markuplint version: `3.5.0`
- Parser lang: Vue
- Node.js version: `18.x`
- OS: macOS or Windows

## Code Example

```html
<template>
  <button @click.stop="foo">...</button>
</template>
```

## The current behavior

`invalid-attr` rule error

## The expected behavior

No error
